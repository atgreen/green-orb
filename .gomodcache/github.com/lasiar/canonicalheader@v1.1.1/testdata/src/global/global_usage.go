package global

func dontImportPackage() {
	header.Get("Test-HEader") // want `non-canonical header "Test-HEader", instead use: "Test-Header"`
}
