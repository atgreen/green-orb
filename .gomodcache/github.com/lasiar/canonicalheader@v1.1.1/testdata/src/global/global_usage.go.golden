package global

func dontImportPackage() {
	header.Get("Test-Header") // want `non-canonical header "Test-HEader", instead use: "Test-Header"`
}
